src/PrestaShopBundle/Resources/config/services/adapter/product/_image.yml

```diff
@@ -51,6 +51,13 @@ services:
       - name: tactician.handler
         command: PrestaShop\PrestaShop\Core\Domain\Product\Image\Query\GetProductImages
 
+  PrestaShop\PrestaShop\Adapter\Product\Image\QueryHandler\GetShopProductImagesHandler:
+    autowire: true
+    public: false
```

Comment:
I wonder will module classes be able to access private services without injecting them? Like if some of them will be needed in legacy controllers (which probably was reachable with `container->get()` up until now?

Reply:
If I'm not mistaken they can use Service locator:
https://symfony.com/doc/4.4/service_container/service_subscribers_locators.html#reusing-a-service-locator-in-multiple-services

Reply:
Yeah, and anyway the handlers shouldn't be used directly so it's fine